vaults/operator-deploy/rbac.yaml

```diff
@@ -46,4 +46,7 @@ roleRef:
 subjects:
   - kind: ServiceAccount
     name: vault
-    namespace: ${ .Namespace }
+    namespace: <VaultKubernetesNameSpace>
```

Comment:
Why you need to have one `vault` ServiceAccount per namespace? isn't it enough to have a serviceAccount at the client (OrchestrateNamespace) with a authorize role at the `<VaultKubernetesNameSpace>` ?